config/config.yaml

```diff
@@ -30,17 +30,18 @@ default:
             peer_password: pw
         cfdp:
             mib:
-                path: ../tmp/mib
-            outgoing:
-                path: ../tmp/outgoing
-            incoming:
-                path: ../tmp/incoming
-            temp:
-                path: ../tmp/temp
-            pdu:
-                path: ../tmp/pdu
-            test:
-                path: ../tmp/test
+                path: ../config/mib
```

Comment:
This can be simplified as `./mib` right?

A few follow questions for you from our previous conversation. Is there a default file of config that the MIB folder should contain? Is this folder only meant to hold the MIB object that we handle writing for the user? If there's no reason a user would need to see / manually edit this file we should just remove this config option I think. Can you elaborate on exactly what it's doing?

Reply:
Yeah duh, you're right. 

There doesn't need to be a default file. When an instance starts up it'll try to load something from file but if the file doesn't exist, it has default values to use. And on shutdown of the instance, it'll persist the config to file. There is no API right now for the user to issue commands to update the MIB, so the way to update it would be to edit the files directly. That whole workflow could become more elegant in the future, but right now this is the way it works.